Formula/go-jira.rb

```diff
@@ -0,0 +1,37 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "f60aa1ae0c9334d49b61adbf0e3b222076f07b975186bcac36d1583999265a5d" => :sierra
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
+    end
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
```

Comment:
These lines can be deleted.

Reply:
ugh. you're right. style guide says to remove them, I just forgot :(

Reply:
No problem 😄